metabase/metabase
# Webhooks Admin Section

Closes https://github.com/metabase/metabase/issues/45574

### Description
Replaces the `Slack` setting in admin with `Notifications`, which allows users to create / update / delete Webhooks.
[Figma Link](https://www.figma.com/design/kYE1YkXvpH8S1DzNZc1YUC/Enable-users-to-create-custom-webhook-notification-channels?node-id=1537-1757&t=Boa1XupAc9iH81XI-0)

### How to verify

1. Go to admin -> notifications
2. Create a new notification (note that a valid URL is needed to accept a post request)
3. Ensure that field validation is working.
4. Update and delete notifications as well.

### Demo

### Checklist

- [s] Tests have been added/updated to cover changes in this PR
